eng/pipelines/templates/jobs/archetype-sdk-client.yml

```diff
@@ -102,7 +103,7 @@ jobs:
           DOTNET_CLI_TELEMETRY_OPTOUT: 1
         inputs:
           filePath: "eng/scripts/CodeChecks.ps1"
-          arguments: -ServiceDirectory ${{parameters.ServiceToTest}}
+          arguments: -ServiceDirectory ${{parameters.ServiceToTest}} -ExcludeProjects "${{ join('|', parameters.ExcludeProjects) }}"
```

Comment:
Tip: for easy future diffing, I recommend using "arguments: >" and putting the arguments on separate lines like we do in a few other places (like calling dotnet).

Reply:
@heaths if the T1 library is deprecated, then could we possibly remove it from the repository entirely? Or is it too early (is servicing still required).

Reply:
Fair enough.   I split them for better readability.